app/components/pages/Login/index.js

```diff
@@ -80,55 +90,69 @@ class LoginPage extends React.Component {
     }
 
     return (
-      <Container mx="auto" width={[1, 410]}>
-        {/* roughly centre the box vertically */}
-        <Box mb={4}>
-          <H2>
-            eLife is changing the way work is reviewed and selected for
-            publication
-          </H2>
-        </Box>
-        <Box mb={5}>
-          <Paragraph.Reading>
-            The leading scientists behind eLife are committed to rapid, fair,
-            and constructive review. Before you submit your work, please note
-            that eLife is a very selective journal that aims to publish work of
-            the highest scientific standards and importance.
-          </Paragraph.Reading>
-          <Paragraph.Reading>
-            Our new manuscript submission system will guide you through the
-            process of submitting your research.
-            {!token && ' Log in with your ORCID identifier to get started.'}
-          </Paragraph.Reading>
-        </Box>
-        {token ? (
-          <Box mb={5}>
-            <ButtonLink data-test-id="continue" primary to="/">
-              Continue
-            </ButtonLink>
-          </Box>
-        ) : (
-          <Flex alignItems="center" mb={5}>
-            <Box mr={3}>
-              <a href={loginUrl}>
-                <ButtonOrcid data-test-id="login">Login with ORCID</ButtonOrcid>
-              </a>
-            </Box>
+      <Box
+        mx={['24px', '24px', '12.5%', '8.333%', '16.666%']}
```

Comment:
We can use reference these values from the theme. You can find the theme [here](https://github.com/elifesciences/elife-xpub/blob/develop/client/elife-theme/src/index.js#L50-L62) and it's usage for spacing [here](https://github.com/elifesciences/elife-xpub/blob/develop/app/components/global/AppBar/AppBar.js#L14).

Reply:
This was changed to be static at all breakpoints in the latest commit but will remember for future reference.